# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/icc/pkg_make_globh.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./dir.c.orig	Fri Jun 21 18:44:17 2002
+++ ./dir.c	Fri Jun 21 18:47:57 2002
@@ -994,6 +994,7 @@
 
 /* Hooks for globbing.  */
 
+#define _FILE_OFFSET_BITS 32
 #include <glob.h>
 
 /* Structure describing state of iterating through a directory hash table.  */
--- ./read.c.orig	Fri Jun 21 18:48:34 2002
+++ ./read.c	Fri Jun 21 18:48:45 2002
@@ -21,6 +21,7 @@
 
 #include <assert.h>
 
+#define _FILE_OFFSET_BITS 32
 #include <glob.h>
 
 #include "dep.h"
